server.ts

```diff
@@ -28,8 +28,12 @@ app.post("/", (req, res) => {
     const moves = req.body.moves;
     const advancedRules = moves && (moves.length < 2 || moves[1].includes('rotate'));
     const noFedCheck = req.body.noFedCheck;
-    const engine = new Engine(moves, {advancedRules, noFedCheck});
+    const engine = new Engine(moves, {advancedRules, noFedCheck: true});
 
+    // reenable fedCheck
+    engine.options.noFedCheck = false;
+
+    // regenerate commands
     engine.generateAvailableCommandsIfNeeded();
```

Comment:
Need to be changed to `engine.gerateAvailableCommands()`, to regerenate commands with fed check off.

Reply:
ok